src/main/webapp/scripts/createGame.js

```diff
@@ -20,3 +35,140 @@ function hideAllExcept(ref) {
     }
   }
 }
+
+// TODO: Find a better way to tell this function what type of question it is
+function addNewQuestion(isMC) {
+  // Sets the header if idMC (is multiple choice) is true or not
+  var header = isMC ? "Multiple Choice" : "True or False";
+
+  let formElement = document.getElementById("createGameForm");
+  let num = formElement.children.length -2; // This is the index for the next question
```

Comment:
This seems like this code could break easily if the HTML structure changes. Is there another way to get this value that is not coupled to the HTML structure?

Reply:
I can have a global variable that stores the amount of questions.